src/vs/workbench/api/common/extHostTerminalService.ts

```diff
@@ -779,7 +780,7 @@ export abstract class BaseExtHostTerminalService extends Disposable implements I
 		});
 	}
 
-	public async $provideTerminalCompletions(id: string, options: ITerminalCompletionContextDto): Promise<vscode.TerminalCompletionItem[] | vscode.TerminalCompletionList | undefined> {
+	public async $provideTerminalCompletions(id: string, options: ITerminalCompletionContextDto): Promise<vscode.TerminalCompletionItem[] | TerminalCompletionList | undefined> {
```

Comment:
It was `extHost.protocol.ts` I was thinking of here, that defines types that go across IPC. We shouldn't be returning objects or API interfaces here.

See `$provideTerminalQuickFixes` as an example:
 https://github.com/microsoft/vscode/blob/7775bb6fc9d99a41560c20562addccf326b14a5f/src/vs/workbench/api/common/extHostTerminalService.ts#L820

Reply:
Ah that makes more sense - I had recalled you said extHostTypes, but was confused bc that is still in browser so would be wrong for remote.